src/Sql/dbo/Tables/ProviderOrganization.sql

```diff
@@ -0,0 +1,11 @@
+﻿CREATE TABLE [dbo].[ProviderOrganization] (
+    [Id]             UNIQUEIDENTIFIER    NOT NULL,
+    [ProviderId]     UNIQUEIDENTIFIER    NOT NULL,
+    [OrganizationId] UNIQUEIDENTIFIER    NULL,
+    [Key]            VARCHAR (MAX)       NULL,
```

Comment:
It would seem prudent to create 1 JSON, `NVARCHAR(MAX)` column here for `ProviderOrganization` in order for future expansion of roles, responsibilities, relationship types and other attributes (such as "who pays what", "max seats allowed for expansion", and other types of future extensions to this relationship between providers and orgs).

Reply:
Not a huge fan of json columns if they can be avoided. Do we know what we want to call it? `Settings` perhaps?

Reply:
I understand, they are harder to query in SQL Server, but they're magical for leaving schema as-is for things that don't need to be queried against (future-proofs DB schema for non-essential fields), only returned back to the middle-tier. `Settings` is a fine name, yes.